sips/sip-029/sip-029-halving-alignment.md

```diff
@@ -0,0 +1,159 @@
+
+Additionally, the increased block production rate under Nakamoto (from 122 to 144 blocks per day) means that without modification, the 2050 supply would exceed the accepted supply cap of 1,818,000,000 STX.
+
+While ultimately halvings do need to happen, for those reasons, it would be highly preferable to not change the economic incentives until both Nakamoto and sBTC have been live for some time, as well as to have reductions in the STX block reward happen at the same time as reductions in the BTC block reward given their links.
+
+Therefore, this SIP proposes altering the token emission schedule to preserve the existing incentive structure while ensuring no increase in the 2050 target supply, and incorporating these principles:
+
+* All STX holders would see a reduced 2050 target STX supply by 0.77%, thereby making STX more scarce
+* All STX holders would see a reduced tail inflation (from 125 STX to 62.5 STX) after the final halving
+* Stacks miners and signers’ economic incentives would remain consistent with the existing incentives for the next 16 months
+* The new Stacks halving schedule would align with Bitcoin halvings starting in 2028 to strengthen the connection the Stacks L2 has to Bitcoin and also synchronize the economic adjustments of both assets, reducing changes in incentives for miners and signers at each halving, as further outlined in the Forum post [3]
+
+# Specification
+
+Applying these upgrades to the Stacks blockchain requires a consensus-breaking network upgrade, in this case, a hard fork. Like other such changes, this will require a new Stacks epoch. In this SIP, we will refer to this new epoch as Stacks 3.1.
+
+The _current_ STX emission schedule is presented as follows.  Note that the **first STX halving is in December 2024**.  The tail emission after the final halving in 2050 would be 125 STX per block, and due to the increased block production rate under Nakamoto (144 blocks per day vs the previous 122), the total supply at that time is projected to be 1,829,119,597 STX, which exceeds the accepted supply cap of 1,818,000,000 STX (0.61% higher).
```

Comment:
Again, I think just remove the stuff about blocks per day since that does not affect the supply. I am not sure why the original estimate was off, but that is not why.

```suggestion
The _current_ STX emission schedule is presented as follows.  Note that the **first STX halving is in December 2024**.  The tail emission after the final halving in 2050 would be 125 STX per block.
```

Reply:
The original estimate was off because it presumed launching Stacks 2.0 with the bitcoin halving in 2020. Instead it was launched in early January 2021 and the schedule wasn't adjusted to to sync up exactly with bitcoin halvings. Halvings are  merely 4, 8 and 12 years after the launch of Stacks 2.0 without the proposed adjustment in SIP-029. @will-corcoran In case you want to add something about the real reason.